modules/motions/hooks/useRegistryWithLimits.ts

```diff
@@ -92,3 +94,30 @@ export function useMapAll({ registryType }: HookArgs) {
   const partners = useAll({ registryType })
   return useMap(partners)
 }
+
+export function useLimits({ registryType }: HookArgs) {
+  const { chainId } = useWeb3()
+  const registry = REGISTRY_WITH_LIMITS_BY_MOTION_TYPE[registryType].useRpc()
+
+  return useSWR(
```

Comment:
Can we get rid of some boilerplate here and in `useAllowedRecipient.ts` by making abstract `useLimits`?